fcli-core/fcli-common/src/main/java/com/fortify/cli/common/variable/FcliVariableHelper.java

```diff
@@ -115,7 +116,7 @@ public static final Writer getVariableContentsWriter(String variableName, String
         checkVariableName(variableName);
         VariableDescriptor descriptor = createVariableDescriptor(variableName, defaultPropertyName, singular, encrypt);
         saveVariableDescriptor(descriptor);
-        PrintWriter pw = new PrintWriter(Files.newOutputStream(getVariableContentsAbsolutePath(variableName), StandardOpenOption.CREATE, StandardOpenOption.WRITE, StandardOpenOption.TRUNCATE_EXISTING));
+        PrintWriter pw = new PrintWriter(new OutputStreamWriter(new FileOutputStream(getVariableContentsAbsolutePath(variableName).toString()), StandardCharsets.UTF_8)) ;
```

Comment:
Apparently we were not using a try-with-resource block before either, but please change to use try-with-resources.